tests/test_unit/test_transfers/transfer_fixtures/transfer_fixture.py

```diff
@@ -46,7 +45,7 @@ async def group_transfer(
 
     queue = await create_queue(
         session=session,
-        name=f"{secrets.token_hex(5)}_test_queue",
+        name=f"test_queue",
```

Comment:
This fixture is used in unit tests. Does it really need fixed queue name?

Reply:
Our worker listens this queue and unit tests do not seem to need it?

Reply:
Unit tests does not use worker. Integration tests do.

Reply:
I see. Did I make my answer unclear?

Reply:
I still don't get it. IMHO it's better to keep unit & integration test entities (including names) separated, to avoid situations when failed integration test lead to failing unit tests (e.g. queue already exists, and wasn't properly deleted), and vice versa

Reply:
That makes sense, I'll add a separate transfer fixture for integration tests then